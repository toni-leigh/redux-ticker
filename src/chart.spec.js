import React from 'react';
import renderer from 'react-test-renderer';

import Chart from './chart';

test('<Chart />', () => {
  const trends = [{ 
    bid: 1,
    timestamp: 3
  }, { 
    bid: 2,
    timestamp: 4
  }];

  expect(renderer.create(
    <Chart trends={ trends }/>
  ).toJSON()).toMatchSnapshot();
});