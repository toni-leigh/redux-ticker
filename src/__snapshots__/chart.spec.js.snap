// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Chart /> 1`] = `
<div
  className="chart"
>
  <div
    className="chart-row chart-row--down"
  >
    <span
      className="chart-row__bid"
    >
      1
    </span>
    <span
      className="chart-row__extra"
    >
      [
      0.0
      , 
      0.0%
      ]
    </span>
    <span
      className="chart-row__arrows"
    >
      
    </span>
  </div>
  <div
    className="chart-row chart-row--up"
  >
    <span
      className="chart-row__bid"
    >
      2
    </span>
    <span
      className="chart-row__extra"
    >
      [
      1.0
      , 
      50%
      ]
    </span>
    <span
      className="chart-row__arrows"
    >
      ↑↑↑
    </span>
  </div>
</div>
`;
